booster_bdd/run.sh

```diff
@@ -2,6 +2,12 @@
 CONFIG=${1:-config}
 CONFIG_FILE="config/$CONFIG.sh"
 
+function prepare_venv() {
+	virtualenv -p python3 venv && source venv/bin/activate && python3 `which pip3` install -r requirements.txt
```

Comment:
Why to use `virtualenv` and not build-in `venv`? 

> virtualenv is a third party alternative (and predecessor) to pyvenv. It allows virtual environments to be used on versions of Python prior to 3.4, which either don’t provide pyvenv at all, or aren’t able to automatically install pip into created environments.

https://docs.python.org/3.4/installing/index.html?highlight=virtualenv

Reply:
Ah, I didn't know it existed... Thanks, I'll use the built-in one instead.